DCT-459 (#1122)
reach-sh/reach-lang: docs/src/quickstart/index.md

$ wsl --set-default-version 2
```

After setting up the default version, install the WSL distribution:

``` cmd
$ wsl --install -d Ubuntu
```

:::note
When the installation is finished, a terminal called `Ubuntu` should open on your screen.
It will request that you set up a user and password.
Make sure you don't skip this step.
:::

## {#qs-installing-docker} Installing Docker:

To install Docker we will use `winget`.
It is the official package manager for Windows.
We will download the package from Microsoft's official GitHub account.

To install Winget run:
``` cmd
cd c:\downloads;
wget -uri https://github.com/microsoft/winget-cli/releases/download/v1.3.431/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle -outfile .\winget.msixbundle; .\winget.msixbundle
```
A window should pop up.
Click Update.

Next, install Docker using winget:

``` cmd
winget install docker.dockerdesktop
```

After installing Docker, reboot:
``` cmd
shutdown -r -t 0
```

## {#qs-win-conf-docker} Configuring Docker:

> Open Docker and wait until it is fully started.

Reviewer: ```suggestion
Open Docker and wait for it to initialize.
```

Click the `Settings` (gear) icon along the top of the Docker app.

> ![Settings icon in Docker`](/quickstart/settings-icon.png)

Reviewer 2: @Craig-Elre a few more edit requests:
Line 104: Verify that `WSL 2.0` and `Use Docker Compose V2` are checked.

Line 109: fix that that

Under *Installation for Ubuntu*
remove commas from:
"The following instructions, from Docker
And then, run: x2
